### PR TITLE
In the Outlook 2016 message list, no longer report associated draft information

### LINKED ISSUE #6219: Outlook 2016: NVDA reads "Associated draft, no associated drafts" for every message
Tested on latest Outlook 2016 from Office 365. @dkager reported that he has the same problem since yesterday with Office 365 as well.

# Steps to reproduce:
1. Open a mail folder (e.g. inbox)
2. Arrow through the message list

# Technical details

For every item in the message list, NVDA does the following:
1. Use Outlooks' native COM interface to query some message attributes (read/unread, attachment, meeting request etc)
2. Do a direct UIA query to loop over the children of the table row, filtered by controlType text

The "Associated draft" column has either been added very recently, or changed it's type to text and so is included in the loop in step 2.

# Solution

I'm not that familiar with Outlook, but I guess this column sometimes contains real information about an associated draft, so just dropping it might not be the solution. We might drop it if it contains the "no associated drafts" value, but this is something that should be translated exactly in all languages and is to error prone. May be Microsoft can revert this change or we can come up with a better way to determine which columns to read.

### PR DESCRIPTION
**Pull request by michaelDCurran**
Fixes #6219

### CODE REVIEW

**Comment by michaelDCurran** [comment]
@jcsteh review please. Very small change, but extremely large impact for Outlook 2016 users. Is top complaint by users to Microsoft Office team

**Comment by michaelDCurran** [comment]
Hoping for 2016.3

File: source/appModules/outlook.py [hunk 1]
```diff
@@ -418,6 +418,11 @@ def _get_name(self):
 		cachedChildren=self.UIAElement.buildUpdatedCache(childrenCacheRequest).getCachedChildren()
 		for index in xrange(cachedChildren.length):
 			e=cachedChildren.getElement(index)
+			# Outlook 2016 started exposing the draft column as a text node.
```
Comment (jcsteh) [inline source/appModules/outlook.py]: nit: Please add issue reference (#6219).

**Comment by jcsteh** [comment]
Otherwise, looks fine to me. Do we want to test this on next briefly or just push straight to master? IMO, more time on master is probably a good thing in this case and it's a tiny change.

**Pull request merged**